testing/describes.js

```diff
@@ -489,19 +480,13 @@ class IntegrationFixture {
   }
 
   /** @override */
-  setup(env) {
+  async setup(env) {
     const body =
       typeof this.spec.body == 'function' ? this.spec.body() : this.spec.body;
     const css =
       typeof this.spec.css == 'function' ? this.spec.css() : this.spec.css;
-    const experiments =
-      this.spec.experiments == undefined
-        ? undefined
-        : this.spec.experiments.join(',');
-    const extensions =
-      this.spec.extensions == undefined
-        ? undefined
-        : this.spec.extensions.join(',');
+    const experiments = this.spec.experiments?.join(',');
+    const extensions = this.spec.extensions?.join(',');
```

Comment:
According to the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), it looks like optional chaining was introduced by Node 14, which enters active LTS [4 weeks from now](https://nodejs.org/en/about/releases/). Until then, we're going to be on [Node 12](https://github.com/ampproject/amphtml/blob/master/.travis.yml#L3-L4).

Would it make sense to leave out this change until then and add a TODO?

Reply:
My bad; I was under the impression it was available for use in build/infra code, just not in runtime code. I'll remove these

Reply:
Done.